Entity/Parameter.php

```diff
@@ -14,7 +17,7 @@
 class Parameter extends BaseParameter
 {
     /**
-     * @ORM\Column(type="string")
+     * @ORM\Column(type="string", unique=true)
```

Comment:
After thinking about it more, we should remove this and consider it in a separate pull request since it would technically be a BC break I think.

Reply:
ok, i removed this as well